org.ppsspp.PPSSPP.json

```diff
@@ -18,69 +15,30 @@
   "modules": [
     "shared-modules/glu/glu-9.json",
     "shared-modules/glew/glew.json",
-    {
-      "name": "libzip",
-      "buildsystem": "cmake-ninja",
-      "config-opts": ["-DCMAKE_BUILD_TYPE=RelWithDebInfo"],
-      "cleanup": [
-        "/bin",
-        "/include",
-        "/lib/pkgconfig",
-        "/share/man"
-      ],
-      "sources": [
-        {
-          "type": "archive",
-          "url": "https://libzip.org/download/libzip-1.7.1.tar.xz",
-          "sha256": "a020dc1b17a79faa748e43a05f8605d65e403813e8e97ad3a300a90ddba097ac"
-        }
-      ]
-    },
     {
       "name": "ppsspp",
-      "buildsystem": "simple",
+      "buildsystem": "cmake",
       "build-options": {
-        "arch":{
+        "arch": {
           "aarch64": {
-            "env": {"EXTRA_OPTS":"-DUSING_GLES2=ON"}
+            "env": {
+              "CC": "clang",
+              "CXX": "clang++"
+            }
           }
         }
       },
-      "build-commands": [
-        "cd ffmpeg && ../ffmpeg-build.sh",
-        "cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCMAKE_INSTALL_PREFIX=${FLATPAK_DEST} ${EXTRA_OPTS} .",
-        "make -j${FLATPAK_BUILDER_N_JOBS} && make install",
-        "ln -s /app/bin/PPSSPPSDL /app/bin/ppsspp",
-        "install -Dm644 icons/icon-512.svg /app/share/icons/hicolor/scalable/apps/ppsspp.svg",
```

Comment:
We need `"rename-icon": "ppsspp"`, as the icon name must be the same as id.   
The ppsspp cmake already installed icons to `/app/share/icons/hicolor/`, so this svg icon is not needed.  
Also need to use `desktop-file-edit  --set-icon="${FLATPAK_ID}" ...` to set desktop file icon name.